extensions/vscode/src/utils/errors.ts

```diff
@@ -59,6 +66,17 @@ export const getSummaryStringFromError = (location: string, error: unknown) => {
     return resolveAgentJsonErrorMsg(error);
   }
 
+  if (isAgentError(error)) {
+    msg = error.msg;
+    if (error.code) {
+      msg += `, Code=${error.code}`;
+    }
+    if (error.operation) {
+      msg += `, Operation=${error.operation}`;
+    }
+    return msg;
+  }
```

Comment:
For my understanding, the changes introduced to `AgentError` in TS files are meant for debugging purposes and do not represent a change in functionality, am I right?

Just to confirm we are not adding up on having cryptic messages showing up to users

Reply:
We've had some additional changes within this area, to remove the cryptic side of the unexpected error reporting which relies upon these updates. The additional functionality you saw above was to improve pulling out any error message we might have within the AgentError responses. This is certainly an area we'll be coming back to, but for now, for this release and PR, we decided to remove the diagnostic information, so that the error messages are brief and further diagnostic info is now directed to the VSCode output pane for Window (by using console.error()).